### DISCUSSION TITLE
StormSim CHS support data files

### DISCUSSION THREAD

**Discussion by jmelby** [coastal-hazards-system/StormSim-Library#8] [General]
74:\work\current\StormSim\CHS_PCHA_Files
Data from regional study - trying to maintain this as most up-to-date version
SRR, Bias and uncertainty file, save point staID file, Storm Parameters, Probability Masses
Need to add a file description

**Comment by fgarcia0128** [comment]
Would be good to also document the dimensions of the datasets and the relationship between each. That could help us come up with a better way to organize/distribute this.

**Reply by jmelby** [reply]
Agree.  Is there a "thumbs-up" in Github?

**Reply by fgarcia0128** [reply]
Little face to the right should be it.

**Reply by jmelby** [reply]
It is on my list to add nodal staID file, nodal PMs and nodal hazards.  Is there anything else required from CHS regional studies for now?

**Reply by fgarcia0128** [reply]
The list of "bad" storms from the regional studies.  So we can do testing on the impact of adjusting the PMs of a partially dry node.

**Reply by jmelby** [reply]
Yup.  More generally, QAQC.  What should that look like?  First cut is to make a QAQC code that looks for all non-values and excessively large values for SWL, Hm0 and Tp and makes a list of "bad" storms.  Madison said Luke has compiled this info.  They are wrapped up with a FEMA deadline but Madison will round it up next week.

**Reply by KHodgens** [reply]
I'm catching up here...did Madison / Luke provide the QAQC info?

**Reply by fgarcia0128** [reply]
There is already a QA/QC process in StormSim that looks for entries with NaN's and/or an excessive value of SWL, Hm0, Tp.  Not sure if this got handed to Jeff.
